packages/graphql/tests/utils/builders/driver-builder.ts

```diff
@@ -65,7 +72,7 @@ export class DriverBuilder extends Builder<Driver, Partial<Driver>> {
                     lastBookmark() {},
                 };
             },
-        } as Driver);
+        } as any);
```

Comment:
Is there a chance of avoiding this any? Maybe we could find a more accurate type for this object?

Reply:
Does it matter though? It is just mocking used for testing?

Reply:
It is not extremely important, but it is preferable to keep the types as close to the original as possible, mainly for ease of testing.